taxonomy/utils.py

```diff
@@ -279,7 +292,7 @@ def get_blacklisted_course_skills(course_key, prefetch_skills=True):
     return qs.all()
 
 
-def get_course_jobs(course_key):
+def get_course_jobs(course_key, product_type):
```

Comment:
We can default product_type to "course" here because get_course_jobs isn't being called in the changes above and it might potentially effect uses of this method in discovery.